sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/OciBlobDescriptor.cs

```diff
@@ -38,7 +38,7 @@ internal static string ComputeDigest(Stream stream)
                 {
                     stream.Position = 0;
```

Comment:
Similar to the comment about accessing stream length, we can't set the stream position if it is not seekable (based on CanSeek property).

Reply:
Actually.. looks like we don't even support non-seekable streams - https://github.com/Azure/azure-sdk-for-net/blob/a64584452f7552864cba37a43deaa0a424992677/sdk/core/Azure.Core/src/RequestContent.cs#L129